integration_tests/env.go

```diff
@@ -15,22 +15,25 @@ import (
 )
 
 const (
-	KubeconfigVar    = "KUBECONFIG"
-	OperatorImageVar = "OPERATOR_IMAGE"
-	HalyardImageVar  = "HALYARD_IMAGE"
-	BucketVar        = "S3_BUCKET"
-	BucketRegionVar  = "S3_BUCKET_REGION"
-
+	KubeconfigVar                        = "KUBECONFIG"
+	OperatorImageVar                     = "OPERATOR_IMAGE"
+	HalyardImageVar                      = "HALYARD_IMAGE"
+	BucketVar                            = "S3_BUCKET"
+	BucketRegionVar                      = "AWS_DEFAULT_REGION"
```

Comment:
We'll rely on an IAM user to use the bucket we provide for tests

Reply:
What permissions does the user have, and how is it created?

Reply:
The account that we are using here was the one that was created when we spin-up the integration-test cluster on the dev account, it was created with terraform. should we create a new one  🤷🏼‍♂️?
if we merge this change the current infrastructure for it will not be needed.